bot/exts/filtering/filtering.py

```diff
@@ -978,7 +978,10 @@ async def _resolve_action(
         result_actions = None
         if actions:
             result_actions = reduce(ActionSettings.union, actions)
-
+            # If the action is a ban, mods don't want to be pinged.
+            if infr_action := result_actions.get("infraction_and_notification"):
+                if infr_action.infraction_type == Infraction.BAN:
+                    del result_actions["mentions"]
```

Comment:
What if there are no mentions?

```suggestion
                    result_actions.pop("mentions", None)
```

Reply:
Fixed!